api/src/certification/session-management/domain/usecases/create-companion-alert.js

```diff
@@ -0,0 +1,19 @@
+import { withTransaction } from '../../../../shared/domain/DomainTransaction.js';
+
+/**
+ * @typedef {import('../usecases/index.js').CertificationCompanionAlertRepository} CertificationCompanionAlertRepository
+ */
+
+export const createCompanionAlert = withTransaction(
+  /**
+   * @param {Object} params
+   * @param {number} params.assessmentId
+   * @param {CertificationCompanionAlertRepository} params.certificationCompanionAlertRepository
+   **/
+  async function ({ assessmentId, certificationCompanionAlertRepository }) {
+    await certificationCompanionAlertRepository.create({ assessmentId });
+  },
+  {
+    isolationLevel: 'serializable',
```

Comment:
Quel est l'apport d'une elevation de l'isolation sur un INSERT, qui n'a pas de dependances (pas de SELECT sur lequel il repose) ? Sauriez-vous me detailler votre reflexion derriere cela afin de comprendre de mon cote ? :)

Reply:
On a changé la manière de faire en gardant le niveau d’isolation par défaut, mais en posant un verrou sur l’assessment lui-même avec un SELECT FOR UPDATE.